### PR TITLE
MetalLB Operator: convert the jobs format

### PR DESCRIPTION
**Pull request by fedepaol**
Using the new documented format to deply the operators, avoid indexes

### CODE REVIEW

**Comment by fedepaol** [comment]
/pj-reharse pull-ci-openshift-metallb-operator-main-operator-e2e

**Comment by fedepaol** [comment]
pj-rehearse pull-ci-openshift-metallb-operator-main-operator-e2e

**Comment by fedepaol** [comment]
/pj-rehearse pull-ci-openshift-metallb-operator-main-operator-e2e

**Comment by fedepaol** [comment]
/pj-rehearse pull-ci-openshift-metallb-operator-main-operator-e2e

**Comment by fedepaol** [comment]
/pj-rehearse pull-ci-openshift-metallb-operator-main-operator-e2e

**Comment by fedepaol** [comment]
/pj-rehearse pull-ci-openshift-metallb-operator-main-operator-e2e

**Comment by fedepaol** [comment]
/pj-rehearse pull-ci-openshift-metallb-operator-main-operator-e2e

**Comment by fedepaol** [comment]
/pj-rehearse pull-ci-openshift-metallb-operator-main-operator-e2e

**Comment by fedepaol** [comment]
/pj-rehearse pull-ci-openshift-metallb-operator-main-operator-e2e

**Comment by fedepaol** [comment]
/pj-reharse pull-ci-openshift-metallb-operator-main-ci-bundle-operatorbundle

**Comment by fedepaol** [comment]
/pj-reharse pull-ci-openshift-metallb-operator-main-ci-bundle-operatorbundle

**Comment by fedepaol** [comment]
/pj-reharse pull-ci-openshift-metallb-operator-main-operator-e2e

**Comment by fedepaol** [comment]
/pj-reharse pull-ci-openshift-metallb-operator-main-operator-e2e

**Comment by fedepaol** [comment]
/pj-reharse pull-ci-openshift-metallb-operator-main-operator-e2e

**Comment by fedepaol** [comment]
pj-rehearse

**Comment by fedepaol** [comment]
/pj-reharse pull-ci-openshift-metallb-operator-main-operator-e2e

**Comment by fedepaol** [comment]
/pj-reharse pull-ci-openshift-metallb-operator-main-operator-e2e

**Comment by fedepaol** [comment]
/pj-reharse pull-ci-openshift-metallb-operator-main-operator-e2e

**Comment by fedepaol** [comment]
/pj-reharse pull-ci-openshift-metallb-operator-main-operator-e2e

**Comment by fedepaol** [comment]
/pj-rehearse pull-ci-openshift-metallb-operator-main-operator-e2e

**Comment by fedepaol** [comment]
/pj-rehearse pull-ci-openshift-metallb-operator-main-metallb-e2e-metal-ipi-ovn

**Comment by fedepaol** [comment]
failures not related to the change.
/pj-rehearse ack

File: ci-operator/config/openshift/metallb-operator/openshift-metallb-operator-main.yaml [hunk 1]
```diff
@@ -72,15 +70,32 @@ resources:
       memory: 200Mi
 tests:
 - as: operator-e2e
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.13"
```
Comment (liornoy) [inline ci-operator/config/openshift/metallb-operator/openshift-metallb-operator-main.yaml:86]: why 4.13 here?

**Comment by zeeke** [comment]
changes looks congruent to doc:
https://docs.ci.openshift.org/docs/how-tos/testing-operator-sdk-operators/#building-operator-bundles

/lgtm

**Comment by liornoy** [comment]
lgtm also

**Comment by fedepaol** [comment]
/pj-reharse pull-ci-openshift-metallb-operator-main-operator-e2e

**Review by ffromani — COMMENTED** [review]
/lgtm

**Pull request merged**